gauge-browser-agent-otel/tests/step_implementation.js

```diff
@@ -0,0 +1,107 @@
+/* globals gauge*/
+"use strict";
+const path = require("path");
+const {
+  openBrowser,
+  write,
+  closeBrowser,
+  screenshot,
+  click,
+  into,
+  textBox,
+  goto,
+  $,
+  evaluate,
+} = require("taiko");
+const assert = require("assert");
+const headless = process.env.headless_chrome.toLowerCase() === "true";
+const config = require("../config");
+const jwtDecode = require("jwt-decode");
+const axios = require("axios");
+
+const decoded = jwtDecode(config.apiToken); // Agent Token
+const baseUrl = decoded["x-sl-server"]; // Base url of the backend
+
+const createTestSession = () => {
+  return axios.post(
+    baseUrl.replace("/api", "/sl-api/v1/test-sessions"), // Public API to start a test session
+    {
+      testStage: "Gauge Tests",
+      bsid: "51721270-ead5-498b-b22f-c3f9861ea44e", // Better set from environment variable or get from window.$Sealights
```

Comment:
The sample code needs to have this coming from an environment variable or according to any other best-practice used by Gauga.